### PR TITLE
Dedent help string in breeze configuration

### PR DESCRIPTION
**Pull request by potiuk**
In Python 3.13 compiler, docstrings are automatically dedented. And while Python 3.13 is not yet officially supported by Airflow for some strange (and unknown) reasons, when you install breeze with uv tools that have Python 3.13 installed, it might get into a state that the docstrings are dedented in Breeze's bytecode and it causes hash calculation for configuration options that have docstring-generated help differs on different installations.

In order to make it consistent - we are now always dedenting the help strings before hash calculation.

See https://github.com/python/cpython/issues/81283

---
**^ Add meaningful description above**
Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/contributing-docs/05_pull_requests.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

**Comment by potiuk** [comment]
Let's see if dedetnting produces the same hashes on CI as well.

**Review by jedcunningham — APPROVED** [review]

**Comment by potiuk** [comment]
Ok. Look like it works nicely :). Weird, weird stuff.

File: dev/breeze/src/airflow_breeze/commands/setup_commands.py [hunk 1]
```diff
@@ -255,8 +256,31 @@ def get_status(file: str):
...
+
+    Python 3.13 automatically dedents docstrings retrieved from functions.
+    See https://github.com/python/cpython/issues/81283
+
+    However, click uses docstrings in the absence of help strings, and we are using click
+    command definition dictionary hash to detect changes in the command definitions, so if the
+    help strings are not dedented, the hash will change.
+
+    That's why we must de-dent all the help strings in the command definition dictionary
+    before we hash it.
```
Comment (amoghrajesh) [inline dev/breeze/src/airflow_breeze/commands/setup_commands.py:271]: Yikes, who would have thought :)

**Review by amoghrajesh — COMMENTED** [review]
Fix looks good!

File: dev/breeze/src/airflow_breeze/commands/setup_commands.py (hunk 1 above)
Comment (potiuk) [reply dev/breeze/src/airflow_breeze/commands/setup_commands.py:271]: Yeah.... But it's about 1st Of November... Helloween. Spooky things happening ..

**Pull request merged**